logical_router_static_route.go

```diff
@@ -115,6 +117,53 @@ func (odbi *ovndb) lrsrDelImp(lr string, ip_prefix string) (*OvnCommand, error)
 	return &OvnCommand{operations, odbi, make([][]map[string]interface{}, len(operations))}, nil
 }
 
+func (odbi *ovndb) lrsrDelExImp(lr string, prefix string, nexthop, policy, outputPort *string) (*OvnCommand, error) {
```

Comment:
Sorry, one more comment. It is better that the lrsrDelImp() just call this function now.

Reply:
fixed